cmake/Platform/Android/LYWrappers_android.cmake

```
@@ -48,6 +48,10 @@ function(ly_apply_debug_strip_options target)
         return()
     endif()
 
+    if (NOT ${LY_STRIP_DEBUG_SYMBOLS})
+        return()
+    endif()
```

This doesn't work if the LY_STRIP_DEBUG_SYMBOLS option is unset.
The cmake variable should not be resolved before passing it to NOT.
```suggestion
    if (NOT LY_STRIP_DEBUG_SYMBOLS)
        return()
    endif()
```